src/utils/bulkUpdateUtils.js

```diff
@@ -2,20 +2,29 @@ import Immutable from 'immutable';
 import _ from 'lodash';
 import SourceVector from 'ol/source/vector';
 import { mapquestApi } from './api';
+import ActionTypes from '../redux/ActionTypes';
+import store from '../redux/store';
 
-export function getElevations(hydrants, handleNewHydrants) {
+const { DATA_IMPORTED } = ActionTypes;
+
+export function getElevations() {
+  const { hydrants } = store.getState().hydrants;
   // make it an array to ensure consistent order
   const needsElevation = hydrants.filter(h => !h.get('elevation')).toArray();
```

Comment:
What happens if I move a hydrant and run getElevations? I assume that hydrants elevation won't be fetched.

Reply:
Good call! I'll make sure we reset elevation to null whenever the coordinates are changed by dragging on the map. 

What do you think about dealing with lat/long changes in the HydrantForm? I don't want to clear it when they change those by default, because maybe they've typed in elevation first and don't want to wipe it when they tweak the coordinates, but on the other hand if they change the coordinates manually and aren't inputting new elevation, would they expect that to automatically update?